poppy/manager/default/ssl_certificate.py

```diff
@@ -430,14 +435,8 @@ def get_certs_by_status(self, status):
         return certs_by_status
 
     def update_certificate_status(self, domain_name, certificate_updates):
-        certificate_old = self.storage.get_certs_by_domain(domain_name)
-        if not certificate_old:
-            raise ValueError(
-                "certificate information not found for {0} ".format(
-                    domain_name
-                )
-            )
 
+        certificate_old = self.storage.get_certs_by_domain(domain_name)
```

Comment:
Does this not need a "try/except"?

Reply:
No. Earlier the code was explicitly throwing a ValueError when there is no certificate found for the given domain name. Now, the method get_certs_by_domain() itself does the same thing implicitly.